tensorflow/python/kernel_tests/BUILD

```diff
@@ -4045,6 +4045,7 @@ cuda_py_test(
     size = "medium",
     srcs = ["tridiagonal_matmul_op_test.py"],
     shard_count = 10,
+    tags = ["no_rocm"], 
```

Comment:
What's the root cause of the UT failure?
cc @deven-amd

Reply:
It is throwing a error on the number of dims for the matrix, saying it's expecting at least 2 but got 1. The specific test is TridiagonalMulOpTest.testInvalidShapesEagerGpu which is supposed to test whether the above error. This is a recently added test it appears. 

This is the offending commit. 
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/commit/68422b215e618df5ad375bcdc6d2052e9fd3080a